plugins/catalog-backend-module-bitbucket-server/src/providers/BitbucketServerEntityProvider.ts

```diff
@@ -37,6 +38,14 @@ import {
   BitbucketServerLocationParser,
   defaultBitbucketServerLocationParser,
 } from './BitbucketServerLocationParser';
+import { Events } from '../lib/index';
+import { EventParams } from '@backstage/plugin-events-node';
+import { CatalogApi } from '@backstage/catalog-client';
+import { TokenManager } from '@backstage/backend-common';
+import { EventBroker } from '@backstage/plugin-events-node';
+
+const DEFAULT_BRANCH = ['master', 'main'];
+const TOPIC_REPO_PUSH = 'bitbucketServer.repo:refs_changed';
```

Comment:
```suggestion
const TOPIC_REPO_REFS_CHANGED = 'bitbucketServer.repo:refs_changed';
```

At other providers, there is a repo push event (`repo:push` at Bitbucket Cloud, `push` at GitHub), however, here it is a different event.